extensions/standard-processors/tests/unit/TailFileTests.cpp

```diff
@@ -41,11 +41,44 @@
 #include "TailFile.h"
 #include "LogAttribute.h"
 
-static std::string NEWLINE_FILE = ""  // NOLINT
-        "one,two,three\n"
-        "four,five,six, seven";
-static const char *TMP_FILE = "minifi-tmpfile.txt";
-static const char *STATE_FILE = "minifi-state-file.txt";
+#ifdef WIN32
+int CR_LEN = 1;
+#else
+int CR_LEN = 0;
+#endif
+
+static std::string FIRST_LINE = "one,two,three";  // NOLINT
```

Comment:
Can you make these const, too? That would also resolve the linter issues here.

Reply:
Even after making it const I still get linter errors:
```
For a static/global string constant, use a C style string instead: "static const char FIRST_LINE[]"
```

Is it ok to leave it as a `const std::string` and with the `// NOLINT` directive?

Reply:
Yes, I'm ok with that.